ZukoAzula2016/src/org/team1540/zukoazula/AutonomousBase.java

```diff
@@ -0,0 +1,159 @@
+    }
+
+    protected void intakeArmRaise(float speed) throws AutonomousModeOverException, InterruptedException {
+        Autonomous.intakeArm.set(Math.abs(speed));
+    }
+
+    protected void waitUntilArmStopped() throws AutonomousModeOverException, InterruptedException {
+        waitUntil(Autonomous.intakeArmStopped);
+    }
+
+    @Override
+    public void loadSettings(TuningContext ctx) {
+        ArrayList<String> settings = new ArrayList<>();
+        for (Field f : this.getClass().getDeclaredFields()) {
+            Tunable annot = f.getAnnotation(Tunable.class);
+            if (annot != null) {
+                f.setAccessible(true);
+                try {
+                    String name = "Auto Mode " + getModeName() + " " + toTitleCase(f.getName()) + " +A";
+                    if (f.getType() == FloatInput.class) {
+                        f.set(this, ctx.getFloat(name, annot.value()));
+                    } else if (f.getType() == BooleanInput.class) {
+                        f.set(this, ctx.getBoolean(name, annot.valueBoolean()));
+                    } else {
+                        Logger.severe("Invalid application of @Tunable to " + f.getType());
+                        continue;
+                    }
+                    settings.add(name);
+                } catch (Exception e) {
+                    Logger.severe("Could not load autonomous configuration for " + this.getClass().getName() + "." + f.getName(), e);
+                }
+            }
+        }
+        Cluck.publishRConf("Auto Mode " + getModeName() + " Settings", new RConfable() {
+            public boolean signalRConf(int field, byte[] data) throws InterruptedException {
+                if (field == 1) {
+                    Autonomous.mainModule.setActiveMode(AutonomousBase.this);
+                    return true;
+                }
+                return false;
+            }
+
+            public Entry[] queryRConf() throws InterruptedException {
+                ArrayList<Entry> entries = new ArrayList<>();
+                entries.add(RConf.title("Settings for " + getModeName()));
+                if (Autonomous.mainModule.getActiveMode() == AutonomousBase.this) {
+                    entries.add(RConf.string("Activate"));
+                } else {
+                    entries.add(RConf.button("Activate"));
+                }
+                for (String setting : settings) {
+                    entries.add(RConf.cluckRef(setting));
+                }
+                entries.add(RConf.autoRefresh(10000));
+                return entries.toArray(new Entry[entries.size()]);
+            }
+        });
+    }
+
+    private String toTitleCase(String name) {
+        StringBuilder sb = new StringBuilder();
+        int lastStart = 0;
+        for (int i = 1; i < name.length(); i++) {
+            if (Character.isUpperCase(name.charAt(i)) || (Character.isDigit(name.charAt(i)) && !Character.isDigit(name.charAt(i - 1)))) {
```

Comment:
I suspect that it's to add spaces between words, including numbers. `addTeam1540` -> `Add Team 1540`